internal/ingress/annotations/loadbalancer/main.go

```diff
@@ -214,18 +216,26 @@ func parseCidrs(ing parser.AnnotationInterface) (out []string, err error) {
 	for _, inboundCidr := range cidrConfig {
 		ip, _, err := net.ParseCIDR(inboundCidr)
 		if err != nil {
-			return out, err
+			return v4CIDRs, v6CIDRs, err
 		}
 
 		if ip.To4() == nil {
-			return out, fmt.Errorf("CIDR must use an IPv4 address: %v", inboundCidr)
+			v6CIDRs = append(v6CIDRs, inboundCidr)
+		} else {
+			v4CIDRs = append(v4CIDRs, inboundCidr)
 		}
-		out = append(out, inboundCidr)
 	}
-	if len(out) == 0 {
-		out = append(out, "0.0.0.0/0")
```

Comment:
would we change the logic to align with V2: https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/v2/pkg/build/lb_security_group.go#L166
Basically only append v4/v6 default settings when customer haven't explicitly configured any cidrs.
If the customer only configured v6 cidr but didn't specify "dualstack", it's typically due an error(where customer expected only v6 works), and we shouldn't append default 0.0.0.0/0 in such case)